.circleci/config.yml

```diff
@@ -384,15 +428,18 @@ workflows:
       - flake8
       - py3_compat
       - fetch_rpms
+      - build_prometheus_rpms
       - build_rpm:
           requires:
             - fetch_rpms
       - install_manager:
           requires:
             - build_rpm
+            - build_prometheus_rpms
```

Comment:
similar to the other comments about putting rpms-into-rpms: make the `build_rpm` job depend on the `build_prometheus_rpms` job, not this one (and not the cluster one)

Reply:
done